validation/src/main/java/se/fortnox/reactivewizard/validation/ValidationModule.java

```diff
@@ -56,4 +54,10 @@ private <T> void bindValidationProxy(Binder binder, Provider<ValidatorUtil> vali
     private <T> Provider<T> validatingProvider(Class<T> iface, Provider<T> wrappedProvider, Provider<ValidatorUtil> validatorUtilProvider) {
         return () -> ValidatingProxy.create(iface, wrappedProvider.get(), validatorUtilProvider.get());
     }
+
+    @Override
+    public Integer getPrio() {
+        // We want this module to override the default binding of ConfigFactory
+        return 110;
```

Comment:
Is there any way of creating a test to validate that this actually works, and more importantly continues to work in the future as well?

Reply:
I added a test that compares priorities between this module and the config module.